CONTRIBUTING.md

```diff
@@ -94,6 +95,16 @@ To run the unit tests, use the following command:
 bun run test
 ```
 
+### E2E Tests
+
+By default, the application is set to serve a web browser in the port `8080` for Playwright. If this port is already being used on your local machine, make sure you change the port in your `.env`.`PLAYWRIGHT_PORT`.
+To run the end-to-end tests, make sure you have latest bundled file before running it. Use the following commands:
+
+```bash
+bun run build
+bun run test:e2e
```

Comment:
don't you have to download the browsers before running this?

Reply:
Yes we do! let me update
Thanks for catching that